source/buildings/sawmill.md

```diff
@@ -22,7 +22,7 @@ layout: default
 ### Note: The Sawmill cannot be built until you have a level 3 [Lumberjack's Hut](../../source/buildings/lumberjack) (or three level 1 Lumberjack's Huts, or another equivalent) and have finished the research in the [University](../../source/buildings/university).
 <br>
 
-The Sawmill is where the Miller will craft any items made of at least 75% wood (no bows or redstone items). For them to do this, you must teach the Sawmill the recipes and the Miller must receive a request for an item from another worker.
+The Sawmill is where the Miller will craft any items made of at least 75% wood (no bows or redstone items). The sawmill will also craft a few specialty items including cactus planks, shingles and timber frames. For them to do this, you must teach the Sawmill the recipes and the Miller must receive a request for an item from another worker.
```

Comment:
Aren't cactus planks, shingles, and timber frames at least 75% wood?

Reply:
Not necessarily. Timber frames are 33% wood in most cases, with shingles being ~50%. Cactus doesn't qualify as wood by default.

Reply:
Then

```suggestion
The Sawmill is where the Miller will craft any items made of at least 75% wood (no bows or redstone items). The Miller will also craft a few other items, including cactus planks, [shingles](../../source/items/shingles), and [timber frames](../../source/items/timberframes). For them to do this, you must teach the Sawmill the recipes and the Miller must receive a request for an item from another worker.
```

Reply:
Also change this on the Miller worker page.